ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/validator/SubmissionValidatorSpec.scala

```diff
@@ -87,14 +87,8 @@ class SubmissionValidatorSpec extends AsyncWordSpec with Matchers with Inside {
       when(mockStateOperations.readState(any[Seq[Bytes]]()))
         .thenReturn(Future.successful(Seq(Some(aStateValue()))))
 
-      def failingProcessSubmission(
-          damlLogEntryId: DamlLogEntryId,
-          recordTime: Timestamp,
-          damlSubmission: DamlSubmission,
-          participantId: ParticipantId,
-          inputState: DamlStateMap
-      ): LogEntryAndState =
-        throw new IllegalArgumentException("Validation failed")
+      val failingProcessSubmission: SubmissionValidator.ProcessSubmission =
```

Comment:
I'd recommend defining in `SubmissionValidator.ProcessSubmission` type aliases that document the meaning of the parameters as we lose the names. Specifically, `RecordTime` and `InputState` should be enough.

Reply:
Sure. 46cbfd4e18219dad344c08641db3f24e3fb10ebc